installation/kubernetes.md

```diff
@@ -22,22 +22,108 @@ LangStream container images are available on the [Github packages registry](http
 * [datastax/langstream-api-gateway](https://github.com/LangStream/langstream/pkgs/container/langstream-api-gateway)
 * [datastax/langstream-cli](https://github.com/LangStream/langstream/pkgs/container/langstream-cli)
 
-### Quick Start
+### Quickstart
 
-To create a LangStream control plane, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster.
+To create a LangStream control plane, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster with a recent version.
 
+Connect to your GCP cluster:
+```bash
+gcloud container clusters get-credentials <langstream-cluster> --region us-east1 --project gcp-techpubs
```

Comment:
what is gcp-techpubs ?

Reply:
Tech docs GCP k8s environment
But it looks distracting, changed in docs